package.json

```diff
@@ -42,7 +43,9 @@
     "mathjs": "^7.1.0",
     "moment": "^2.27.0",
     "node-fetch": "^2.6.0",
+    "patch-package": "^6.2.2",
     "pm2": "^4.4.0",
+    "postinstall-postinstall": "^2.1.0",
```

Comment:
`patch-package` and `postinstall-postinstall` can be dev dependencies, no?

Reply:
no